packages/dlm/dlm.0.3.0/opam

```diff
@@ -13,22 +13,22 @@ build: [
 depends: [
   "ocaml"
   "jbuilder" {>= "1.0+beta10"}
-  "ctypes" {>= "0.11.1"}
-  "lwt"
-  "unix-errno"
-  "ounit"
-  "uuidm"
+  "conf-pkg-config"
+  "ctypes" {>= "0.11.1" & < "0.18.0"}
+  "lwt" {>= "4.2.0"}
+  "unix-errno" {< "0.6.0"}
+  "ounit" {>= "2.0.0"}
+  "uuidm" {>= "0.9.6"}
 ]
 depexts: [
   ["libdlm-dev"] {os-family = "debian"}
   ["dlm-devel"] {os-distribution = "rhel"}
   ["dlm-devel"] {os-distribution = "centos"}
   ["dlm-devel"] {os-distribution = "fedora"}
   ["dlm-devel"] {os-distribution = "ol"}
-  ["dlm-git"] {os-distribution = "arch"}
   ["libdlm-devel"] {os-family = "suse"}
 ]
-available: [ os = "linux" & os != "alpine" ]
+available: [ os = "linux" & os != "alpine" & os != "archlinux"]
```

Comment:
```suggestion
available: [ os = "linux" ]
```
Is it possible to install the package by manually compiling the appropriate library?

Reply:
See above